flavors/bitcoin-signet/config.env

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+export SITE_TITLE='Bitcoin Signet Explorer'
+export HOME_TITLE='Bitcoin Signet Explorer'
+export NATIVE_ASSET_LABEL=rBTC
```

Comment:
We currently use `sBTC` on our side, FWIW.

Reply:
Yes, this should've been sBTC, thanks for the heads up! Fixed in https://github.com/Blockstream/esplora/pull/291/commits/51ded72777cab30a3bab127d3ec8f3742e9641dc.